CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing
+
+Although we are always happy to make improvements, we also
+welcome changes and improvements from the community!
+
+Have a fix for a problem you've been running into or an idea for a new feature
+you think would be useful? Here's what you need to do:
+
+1. [Read and understand the Code of Conduct](#code-of-conduct).
+2. Fork this repo (unless you work at ackama) and clone that repo to somewhere on your machine.
+3. [Ensure that you have a working environment](#setting-up-your-environment).
+4. Read up on the [architecture of the project](#architecture),
+   [how to run tests](#tests), and [the code style we use in this project](#code-style).
+5. Cut a new branch and write a failing test for the feature or bugfix you plan
+   on implementing.
+6. [Make sure your branch is well managed as you go along](#managing-your-branch).
+7. If this branch fixes an issue, use github's `fixes: #issue_number`
+   when writing your commit message to link the issue to your pr.
+8. Push to your fork and create a pull request.
+9. Ensure that the test suite passes CI and make any necessary changes to your branch to bring it to green.
+
+We will review pull requests fortnightly at our ackama ruby guild meeting.
+Once we look at your pull request, we may give you feedback. For instance,
+we may suggest some changes
+to make to your code to fit within the project style or discuss alternate ways of
+addressing the issue in question. Assuming we're happy with everything, we'll
+then bring your changes into main. Now you're a contributor!
+
+## Code of Conduct
+
+If this is your first time contributing, please read the [Code of Conduct]. We
+want to create a space in which everyone is allowed to contribute, and we
+enforce the policies outline in this document.
+
+[Code of Conduct]: https://github.com/ackama/rails-template/blob/main/CODE_OF_CONDUCT.md
+
+## Setting up your environment
+
+1. Install [the most recent ruby version](https://www.ruby-lang.org/en/downloads/) with the ruby version manager of your choice
+2. Run `gem install rails` to get the most recent rails release
+3. Install the node version defined in the .node-version file (with nvm if you prefer)
+4. Install yarn v1.21.0 or later
```

Comment:
🐘 I feel like "or later" could arguably mean "v2" which we definitely do not want to use - maybe expand on this?

Reply:
i got this directly from the README.md, do we want to change it in both places to say `Install yarn v1.21.0 or later but less than v2` (i haven't paid attention to yarn)

Reply:
it should actually be enforced now by `yarn` because it's in the `package.json` - I think just replace it with "install `yarn` v1" because that should be fine